frappe/public/js/frappe/router.js

```diff
@@ -386,6 +386,8 @@ frappe.router = {
 	set_route_options_from_url(route) {
 		// set query parameters as frappe.route_options
 		var last_part = route[route.length - 1];
+		// add routing v2 compatability
+		if (!last_part.includes("?")) last_part = route[route.length - 1] + window.location.search;
```

Comment:
Instead can we set the route as `(window.location.pathname + window.location.search)`? In `get_sub_path_string`

Reply:
`window.location.pathname` will also have `/app` prepended